### PR TITLE
 Validate Schema.json

### PR DESCRIPTION
**Pull request by hpsing**
@albertoabellagarcia can you please check the schema.json either it is correct or not?

and auto schema generator giving empty response:

https://smartdatamodels.org/extra/ngsi-ld_generator_v0.92.php?schemaUrl=https://raw.githubusercontent.com/singhhp1069/dataModel.COVID19/master/VaccinationCertificate/schema.json

can you please check.

### CODE REVIEW

File: VaccinationCertificate/ADOPTERS.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+description: This is a compilation list of the current adopters of the data model [Data model] of the Subject [Subject].  All fields are non mandatory. More info at https://smart-data-models.github.io/data-models/templates/dataModel/CURRENT_ADOPTERS.yaml
```
Comment (albertoabellagarcia) [inline VaccinationCertificate/ADOPTERS.yaml:1]: could we mention here any project?

**Review by albertoabellagarcia — COMMENTED** [review]
Nice, please populate the examples

File: VaccinationCertificate/ADOPTERS.yaml (hunk 1 above)
Comment (hpsing) [reply VaccinationCertificate/ADOPTERS.yaml:1]: there isn't any project but created this for the paper.

**Review by albertoabellagarcia — COMMENTED** [review]
include examples

**Comment by hpsing** [comment]
Example Added.

**Review by albertoabellagarcia — APPROVED** [review]
some minor issues I'll fix after aprove

**Pull request closed without merging**